lib/senders/index.js

```diff
@@ -4,16 +4,15 @@
 
 'use strict'
 
-var P = require('../promise')
 // This indirection exists to accommodate different config properties
 // in the old auth mailer. If/when the two config files are merged and
 // there's nothing left that imports mailer/config, it is safe to merge
 // raw.js and this file into one. Be careful not to mix the arguments
```

Comment:
See https://github.com/mozilla/fxa-auth-server/pull/1681#discussion_r103258203, this was my fault and there's fix for it waiting to be reviewed in another PR.